src/api/index.ts

```diff
@@ -142,17 +142,34 @@ export const getProductsUpdateConsentLink = (
   return `${TINK_LINK_URL}/1.0/products/update-consent?${params.join('&')}`;
 };
 
+export const getAccountCheckUpdateConsentLink = (
+  authorizationCode: string,
+  userId: string,
+  credentialsId: string
+) => {
+  const params = [
+    `client_id=${process.env.REACT_APP_TINK_LINK_PERMANENT_USERS_CLIENT_ID}`,
+    `products=${PRODUCTS}`,
```

Comment:
```suggestion
```

Reply:
This line not needed for `getAccountCheckUpdateConsentLink`

Reply:
fixed in https://github.com/tink-ab/tink-link-web-permanent-users-example/pull/92/commits/2d6eb7b1ba7842542ec04b302f47e7cc98dacf3a